examples/basic_i2c_example.py

```diff
@@ -8,54 +8,63 @@ def main():
     i2c = device.create_interface("i2c")
     print(info)
 
-    print("Initializing the bus...\n")
+    print("Initializing the bus...")
     (success, _) = i2c.init_bus(3300)
     if not success:
         print("I couldn't initialize the bus. Are you sure there's any target connected?")
+        print("Closing the Supernova...")
+        device.close()
         exit(1)
 
+    print("Setting the frequency to 400kHz...")
     success, result = i2c.set_parameters(400000)
-    if success:
-        print(f"Frequency set in: {result} Hz")
-    else:
+    if not success:
```

Comment:
Is ok to not exit if the frequency couldn't be set?

Reply:
done